lib/client.ml

```diff
@@ -1,29 +1,31 @@
-open Lwt.Syntax
 open Stdint
 
 type t =
-  { in_ch : Lwt_io.input_channel
-  ; out_ch : Lwt_io.output_channel
+  { flow :
```

Comment:
This can probably just be `Eio.Flow.two_way` or `Eio_unix.socket`

Reply:
Thank you! Fixed with this casting https://github.com/fraidev/tornado/commit/b2ce02fa69291c59c489d0de98c22c01a4153871